numba/tests/test_np_functions.py

```diff
@@ -3181,6 +3193,155 @@ def test_split_basic(self):
         self.assertIn("np.split: Argument axis out of bounds",
                       str(raises.exception))
 
+    def test_vsplit_basic(self):
+        # split and array_split have more comprehensive tests of splitting.
+        # only do simple test on vsplit
+        def inputs():
+            yield np.array([[1, 2, 3, 4], [1, 2, 3, 4]]), 2
+            yield np.array([[1., 2., 3., 4.], [1., 2., 3., 4.]]), 2
+            yield np.arange(16.0).reshape(4, 4), 2
+            yield np.arange(16.0).reshape(4, 4), np.array([3, 6])
+            yield np.arange(16.0).reshape(4, 4), [3, 6]
+            yield np.arange(16.0).reshape(4, 4), (3, 6)
+            yield np.arange(8.0).reshape(2, 2, 2), 2
+
```

Comment:
Are these tests from the NumPy test suite? If yes, add a link referencing the NumPy GitHub repo